borgmatic/commands/borgmatic.py

```diff
@@ -269,6 +269,12 @@ def run_actions(
         'repositories': ','.join([repo['path'] for repo in location['repositories']]),
     }
 
+    try:
+        log_file = global_arguments.log_file
+        hook_context['log_file'] = log_file
+    except AttributeError:
```

Comment:
I don't think this will actually ever raise because `log_file` is always an attribute on `global_arguments` even if the value is `None`. So an `if` instead of a `try`/`except` might work just fine. Although.. I'd be fine unconditionally setting this into `hook_context` when it's originally assigned above, maybe empty string when the `log_file` value is `None`...?

Reply:
Ah, I got the `AttributeError` from tests, but again, I should've updated the tests only.